From: David Faure <faure@kde.org>
Date: Thu, 09 Jan 2014 19:41:48 +0000
Subject: modem.cpp:169:1: error: no return statement in function returning non-void [-Werror=return-type]
X-Git-Tag: v1.0.1
X-Git-Url: http://quickgit.kde.org/?p=libmm-qt.git&a=commitdiff&h=1d7f3e5d5eb4bb4fba741736049c229b48b6815c
---
modem.cpp:169:1: error: no return statement in function returning non-void [-Werror=return-type]
---


--- a/modem.cpp
+++ b/modem.cpp
@@ -165,7 +165,7 @@
 QString ModemManager::Modem::command(const QString &cmd, uint timeout)
 {
     Q_D(Modem);
-    d->modemIface.Command(cmd, timeout);
+    return d->modemIface.Command(cmd, timeout);
 }
 
 QString ModemManager::Modem::simPath() const

